common/src/test/java/com/revenuecat/purchases/common/DeviceCacheTest.kt

```diff
@@ -442,6 +442,16 @@ class DeviceCacheTest {
         assertThat(sentTokens).isEmpty()
     }
 
+    @Test
+    fun `If getting all preferences throws NullPointerException when calling findKeysThatStartWith, an empty set is returned`() {
+        every {
+            mockPrefs.all
+        } throws NullPointerException("NullPointerException")
+
+        val returnedSetOfKeys = cache.findKeysThatStartWith("any_cache_key")
+        assertThat(returnedSetOfKeys).isEmpty()
+    }
+
```

Comment:
Not sure why this is showing as a change, since it's already in `develop`. I guess it's because the commit has been cherry picked...